### DISCUSSION TITLE
How to get all environment variables？

### DISCUSSION THREAD

**Discussion by kadifu** [shinsenter/php#125] [Q&A]
Hello, I use the tag fpm-apache mirror. I can see in the documentation that you can use 'php-envvars' to view PHP-related environment variables. But what command can I use to view all environment variables, such as Apache-related configurations?

**Comment by shinsenter** [comment]
@kadifu 
You can use the built-in Linux command `printenv` to list all environment variables.
https://man.freebsd.org/cgi/man.cgi?query=printenv

To run it in your container from the host machine:
```
docker exec -it <your_container_id> printenv
```
